tests/Checker/AJAX_Runner_Tests.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Tests for the Checks class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\AJAX_Runner;
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+class AJAX_Runner_Tests extends WP_UnitTestCase {
+
+	public function test_is_plugin_check_returns_true() {
+		// Mock the AJAX request.
+		$_SERVER['REQUEST_URI'] = '/admin-ajax.php?action=plugin_check_run_checks';
```

Comment:
See above: Instead of this (or in addition to this), you'd need to use the `wp_doing_ajax` filter to force it to `true`.